alicloud/resource_alicloud_instance.go

```diff
@@ -1160,27 +1161,49 @@ func modifyInstanceType(d *schema.ResourceData, meta interface{}, run bool) (boo
 			return update, err
 		}
 
-		d.SetPartial("instance_type")
-
-		//An instance that was successfully modified once cannot be modified again within 5 minutes.
-		args := ecs.CreateModifyInstanceSpecRequest()
-		args.InstanceId = d.Id()
-		args.InstanceType = d.Get("instance_type").(string)
-		args.ClientToken = buildClientToken(args.GetActionName())
-
-		err = resource.Retry(6*time.Minute, func() *resource.RetryError {
-			_, err = client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
-				return ecsClient.ModifyInstanceSpec(args)
+		instanceChargeType := d.Get("instance_charge_type").(string)
+		switch {
+		case instanceChargeType == string(common.PrePaid):
+			args := ecs.CreateModifyPrepayInstanceSpecRequest()
+			args.InstanceId = d.Id()
+			args.InstanceType = d.Get("instance_type").(string)
+
+			err = resource.Retry(6*time.Minute, func() *resource.RetryError {
+				_, err := client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
+					return ecsClient.ModifyPrepayInstanceSpec(args)
+				})
+				if err != nil {
+					if IsExceptedError(err, EcsThrottling) {
+						time.Sleep(10 * time.Second)
```

Comment:
I think 10 seconds is too long. 3 or 5 seconds is ok.

Reply:
done